sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -290,6 +290,9 @@ void _nib_nc_get(const _nib_onl_entry_t *node, gnrc_ipv6_nib_nc_t *nce)
             return;
         }
     }
+#else
+    /* Prevent unused function error thrown by clang */
+    (void)_get_l2addr_from_ipv6;
```

Comment:
Can you put the `_get_l2addr_from_ipv6()` function into `GNRC_IPV6_NIB_CONF_6LN` conditional as well, instead, please?

Reply:
OK. I reallize now that this this is not possible, since it is also used in the else case of the `GNRC_IPV6_NIB_CONF_ARSM` case. Then let's go with this.